Add LocalHost

alpinus4 opened the pull request.

alpinus4 committed:
Add blank LocalHost and changing host option in args

blazej-smorawski commented on src/instance_host/local_host.rs:
```diff
@@ -0,0 +1,34 @@
+        Command::new("sh")
+                .arg("-c")
+                .arg("echo hello")
+                .output()
```
I guess `spawn` instead of `output` is the way to go. We should also store the `Child` objects returned by `spawn`, so later on we can close each process.

alpinus4 replied:
Yeah, good point

alpinus4 committed:
Basic running process in local instance host

alpinus4 committed:
Run scene host through podman in local instance host

alpinus4 committed:
Merge branch 'master' into add-local-host

alpinus4 committed:
Fix merge

alpinus4 committed:
Use app_path parameter, run scene host through uvicorn

Proxy not working, TODO
Also TODO add Dockerfile actions to have scene host locally

alpinus4 committed:
Make proxy working with localhost

alpinus4 committed:
Add scene-host to docker image

alpinus4 committed:
Move TODOs

alpinus4 committed:
Remove hardcoded port, it will be inserted two times

alpinus4 committed:
Merge branch 'master' into add-local-host
